src/lib/utils/existsNamespace.ts

```diff
@@ -0,0 +1,16 @@
+import { existsSync } from "fs"
+import { join } from "path"
+
+/**
+ * Checks if a namespace .json file exists for the given locale
+ * @param locale Path locale
+ * @param namespace Namespace for the page
+ * @returns false if the namespace .json file exists, true otherwise
+ */
+export const existsNamespace = (
+  locale: string,
+  namespace: string,
+): boolean => {
+  const nsJsonPathForLocale = join("../intl", locale, namespace + ".json")
+  return !existsSync(nsJsonPathForLocale)
```

Comment:
Sorry, should have detailed better, this should be positive now.
```suggestion
  return existsSync(nsJsonPathForLocale)
```

Reply:
Oops, my bad, meant to get that, good catch